static/app/views/starfish/modules/databaseModule/databaseChartView.tsx

```diff
@@ -75,19 +75,21 @@ export default function APIModuleView({}: Props) {
   const data = Object.values(seriesByAction);
 
   const seriesByQuery: {[action: string]: Series} = {};
-  topGraphData.forEach(datum => {
-    seriesByQuery[datum.description] = {
-      seriesName: datum.description.substring(0, 50),
-      data: [],
-    };
-  });
+  if (!isTopGraphLoading) {
```

Comment:
Also added this conditonal, which kept giving me occasional runtime errors when trying process data that's not loading.